app/models/container_group.rb

```diff
@@ -70,6 +70,14 @@ def event_where_clause(assoc = :ems_events)
     end
   end
 
+  def tenant_identity
+    if ext_management_system
+      ext_management_system.tenant_identity
```

Comment:
In practice, this is the branch that runs.  Neither {Kubernetes,Openshift}::ContainerManager implement it,
falling back on ExtManagementSystem default implementation:
```
  def tenant_identity
    User.super_admin.tap { |u| u.current_group = tenant.default_miq_group }
  end
```